server/routes/allCompletedReports/allCompletedReports.ts

```diff
@@ -46,7 +46,9 @@ export default class AllCompletedReportsRoutes {
       const filter = filterFromUiFilter(uiFilter)
       const results = await this.reportedAdjudicationsService.getAllCompletedAdjudications(
         res.locals.user,
-        filter,
+        /* eslint-disable */
+        filter as any, // TODO - NN-4461
+        /* eslint-enable */
```

Comment:
We really shouldn't be disabling eslint for this - it's there for a reason! The only one we should use is the multiple classes per file one I think
Having to cast `filter` as any means the type needs updating somewhere else

Reply:
I think you need to change the type in this function `FilterFromUiFilter` in the helper file
`status: filter.status || allStatuses`